source/lib/uploads/scripts/druid/terminate_druid_node.sh

```diff
@@ -109,6 +109,8 @@ waitForProcess() {
         else
             echo "The new node is up. Stopping old node..."
             $SUPERVISORCTL_CMD stop $process_name
+            # wait gracefulShutdownTimeout for 30 seconds
+            sleep 30
```

Comment:
Can we pass in `stopwaitsecs` from node config into this script instead of hardcoding it here?

Reply:
Yes that's a good idea, eventually we should avoid hard code, make it configurable and have the flexibility to handle shorter or longer graceful shutdown periods.

I guess the priority and purpose of this PR is to ensure we don't throw 500s during deployment, I'm happy to extend this to make it flexible later.